webpack/scenes/Hosts/ChangeContentSource/helpers.js

```diff
@@ -1,6 +1,11 @@
 import { STATUS } from 'foremanReact/constants';
```

Comment:
Not strictly in scope here, but I notice the 'change content source' page doesn't have the hostname anywhere which is a bit confusing. If it's easy I would love to add it here, otherwise should probably add a Redmine for it.

Reply:
I created redmine for it: https://projects.theforeman.org/issues/34892